tools/depends/target/ffmpeg/CMakeLists.txt

```diff
@@ -37,6 +56,13 @@ elseif(CORE_SYSTEM_NAME STREQUAL osx)
                           --disable-decoder=mpeg_xvmc --disable-vda --disable-crystalhd --enable-videotoolbox
                           --target-os=darwin
                           --disable-securetransport)
+elseif(CORE_SYSTEM_NAME STREQUAL rbpi)
+  if(CPU STREQUAL "cortex-a7" OR CPU STREQUAL "cortex-a53")
+    list(APPEND ffmpeg_conf --cpu=${CPU} --enable-pic --disable-armv5te --disable-armv6t2)
+  endif()
+  list(APPEND ffmpeg_conf --enable-hardcoded-tables --disable-vaapi --disable-vdpau --enable-mmal
+                          --enable-omx-rpi)
+
```

Comment:
It would be best if you could build with and without ``hardcoded-tables`` and report any measured change in CPU usage or startup time of file playback (it may be measurable from debug log messages close to the "Opening:" message).
I think without a measurable performance improvement, and given it increases library size, then it's best to leave it in the default state as disabled.
But if you could show it saves something noticeable (e.g. 500ms) then we may want it.